website/project/model.py

```diff
@@ -1843,12 +1843,12 @@ def get_aggregate_logs_query(self, auth):
         ids = [self._id] + [n._id
                             for n in self.get_descendants_recursive()
                             if n.can_view(auth)]
-        query = Q('__backrefs.logged.node.logs', 'in', ids) & Q('should_hide', 'ne', True)
+        query = Q('node', 'in', ids) & Q('should_hide', 'ne', True)
         return query
 
     def get_aggregate_logs_queryset(self, auth):
         query = self.get_aggregate_logs_query(auth)
-        return NodeLog.find(query).sort('-_id')
+        return NodeLog.find(query).sort('-date')
```

Comment:
I don't know what this means.  I already have these indices - 
```
    __indices__ = [
        {
            'key_or_list': [
                ('user', 1),
                ('node', 1)
            ],
        }
    ]
```
Can I add them here?
```
    __indices__ = [
        {
            'key_or_list': [
                ('user', 1),
                ('node', 1),
                ('should_hide', 1),
                ('-date', 1)

            ],
        }
    ]
```